playbooks/common/openshift-docker/config.yml

```diff
@@ -0,0 +1,85 @@
+- name: Set facts necessary to configure docker
+  hosts:
+    - masters
+    - nodes
+  roles:
+  - openshift_facts
+  tasks:
+  - openshift_facts:
+      role: "{{ item.role }}"
+      local_facts: "{{ item.local_facts }}"
+    with_items:
+      - role: common
+        local_facts:
+          deployment_type: "{{ openshift_deployment_type }}"
+
+- name: Configure docker for additional repositories
+  hosts:
+    - masters
+    - nodes
+  handlers:
+    - name: restart docker
+      service: name=docker state=restarted
+  tasks:
+    # TODO -- currently this is sane to do on masters since we require
+    # that a master be part of the sdn which requires node and docker packages
+    # When we fix the SDN to work without a running node we should clean this up
+    - yum: name=docker state=installed
```

Comment:
That has to be guarded by a is_atomic conditional. Because there are many of those throughout the codebase, do we want a (local) `rpm -q` plugin? It feels cleaner.